_content/essays/buddhism-and-the-incorruptible-body_dhammika.md

```diff
@@ -0,0 +1,20 @@
+---
+title: "Buddhism and the Incorruptible Body"
+authors:
+  - dhammika
+external_url: "http://budblooms.org/2020/05/10/buddhism-and-the-incorruptible-body/"
+source_url: ""
```

Comment:
Bhante first wrote on this topic [in 2009](http://sdhammika.blogspot.com/2009/02/i-want-my-mummy.html) but this essay appears to be new, so let's just remove this field